hydrant/models/datetime.py

```diff
@@ -2,13 +2,19 @@
 from flask import current_app
 from dateutil import parser
 
+WRAP_YEAR = 2025
+
 
 def parse_datetime(data, error_subject=None, none_safe=False):
     """Parse input string to generate a UTC datetime instance
 
     NB - date must be more recent than year 1900 or a ValueError
     will be raised.  (library limitation)
 
+    WRAP_YEAR (defined above) is used to determine how a 2 digit
```

Comment:
I wonder if we can work with each site to get 4-digit years

Reply:
Lydia is on that exact tact.  I really don''t like this guessing / hack.  Would be better to insist on a 4 digit year format.